vk_generator/src/generator/mod.rs

```diff
@@ -29,17 +30,17 @@ pub struct GenConfig<'a> {
 
 impl<'a> GenConfig<'a> {
     /// Create a new generator config. Is identical to `Default::default()`.
-    pub fn new() -> GenConfig<'a> {
+    pub fn new() -> Self {
         Default::default()
     }
 
-    /// Whether or not to remove the `Vk` prefix on structs, enums, and typedefs.
+    /// Whether or not to remove the `Vk` prefix on structs, enums, consts, and typedefs.
```

Comment:
I'd rather not have `remove_type_prefix` control enum variant prefix stripping, because the enum variants themselves aren't types, per say - they're values of a type. I think it would make more sense to control how much prefix stripping is done with the `remove_variant_padding` variable, which already exists to control variant padding.

The best solution I have for that is to change the type of `remove_variant_padding` into an enum, which sets the amount of padding stripping done to one of three levels: no stripping, `VK_` stripping, and total pattern stripping (what's currently done when it's set to `true`).